### DISCUSSION TITLE
doubt about Change Notice

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#554] [Q&A] [answered]
Hi Lab
reading [eForms FAQ](https://docs.ted.europa.eu/home/eforms/FAQ/index.html) I got a doubt about this sentence : 

A Change notice must contain all the information reported in the initial notice, with changes applied, as well as a section describing the latest changes (to the immediately preceding Notice):

Let's take for example a notice subType 4 , I got from DB a previously published notice and  added a Change Section.
Let's look Section Part  : 

![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/cae09180-41a0-48fb-bc4e-59773f9767ce)

Suppose I want to change the TITLE content to "My New Title" .

Should I just change this content or build another field TITLE with the new content ?

of course I filled the Change Section : 

![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/084349e3-e00b-4568-b2ba-759a19356dee)

Thanks and Regards
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
Just update the content of the field, do not add new ones. The generated XML still needs to satisfy the rules and there shall not be more than one occurrence of a given text for a given language.

**Comment by rengit62** [comment]
Hi Lab,
still a question about change notice. ( with sdk 1.7 of course )
In FAQ it is written : 
"**With eForms, there will be a Change notice, which is a reproduction of its parent notice with an extra section to advertise changes to the procurement and procurement documents and for correction of clerical errors. Major changes such as adding or removing Lots to a published Contract Notice cannot be done through a Change notice; in this case, a new CN would be expected**."
I guess that cannot be done also adding or removing Part, Group of Lots, and .. others ? For example adding/removing a Buyer, Organization,..
Besides, I found this : 
"**Only three fields have the new property inChangeNotice. Will it be added to all other fields? Can a field without the property never or always be changed?**"
in fields.json I found 2 fields with this property : 
"id" : "BT-1375-Procedure" and  "id" : "BT-262-Procedure", where is the third ?
Bye
Renato

**Reply by YvesJo** [reply]
Hi Renato,
In SDK 1.7, only 2 fields have this property. This initial implementation for change controls needed a review and you may expect some updates. Dynamic rules will be available in a next release as part of the stage 6b of these rules.
